install-dependencies

```diff
@@ -0,0 +1,71 @@
+#!/usr/bin/env bash
+
+# Clean object dir and return the input error
+function err(){
+  rm -rf $DEPS_GLIBC
+  exit $1
+}
+
+#
+# Applies OSX "dependencies" needed to compile the linux kernel
+#
+
+if [[ $(uname -s) = "Darwin" ]]; then
+  command -v port >/dev/null ||
+  (
+    echo "MacPorts is not installed" &&
+    echo "Please visit https://www.macports.org/ for more information" &&
+    exit 40
+  )
+
+  port -q install gsed || err $?
+
+  ln -s /opt/local/bin/gsed /usr/local/bin/sed
```

Comment:
Should I put the symlink in the project dir instead and just modify the user's PATH to point to it, or is this alright?